ext/EffectsGLMExt.jl

```diff
@@ -0,0 +1,14 @@
+module EffectsGLMExt
+
+using Effects
+
+using GLM: AbstractGLM, Link, Link01, inverselink
+using StatsModels: TableRegressionModel
```

Comment:
IIUC, extensions are supposed to bring in package dependencies by prefixing with the package. I haven't seen that documented anywhere but I've seen analogous suggestions made elsewhere.
```suggestion
using Effects.StatsModels: TableRegressionModel
```